create_limo_profile.py

```diff
@@ -0,0 +1,21 @@
+"""
+This script is used to generate a new api key for the limo profile for the limonade service.
+"""
+
+import requests
+import random
+
+def main():
+    headers = {'Authorization':'Bearer admin'}
+    random_id = random.randint(0, 10000000)
+    response = requests.post('http://localhost:9000/v1/profiles', json={'name': 'limo', 'email':f'limo{random_id}@dourolabs.com','role':'protocol'}, headers=headers)
```

Comment:
What will happen if we create the same profile multiple times? I think it'll raise an error which says email already exists. Should we catch this here? I think it's not gonna raise error if we create access_token for the profile twice.